### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by SHtress**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,87 +1,107 @@
...
 cd ostis-example-app/scripts
 ./run_sc_server.sh
+```
+
+After that launch sc-web interface:
 
-#Terminal 2
+```sh
 cd ostis-example-app/scripts
 ./run_scweb.sh
```
Comment (AlexandrZagorskiy) [inline README.md]:
```suggestion
./run_sc_web.sh
```

File: README.md [hunk 2]
```diff
@@ -1,87 +1,107 @@
...
 
+`problem-solver` is the place for the problem solver of your app. Put your agents here. After changes in problem-solver you should rebuild it:
 
-#### Agents on C++
-Some tips:
-- Store your modules with C++ agents in *problem-solver/cxx*;
-- After updating your C++ code you need to rebuild problem-solver. Just run:  
-```
+*Use **scp_stable** branch for the development of agents on SCP.*  
+*Use **0.5.0**, **0.6.0** or **0.7.0** branch for the development of agents on C++. You can see an example module with a C++ agent [here](problem-solver/cxx/exampleModule/README.md)*
```
Comment (AlexandrZagorskiy) [inline README.md:59]: Should we mention here previous versions? If we should, can you mention that last version is much more stable?

**Review by AlexandrZagorskiy — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (SHtress) [reply README.md:59]: Which versions? scp_stable(0.4.0) and other are release versions and can be used. Last version is often more stable, not?

**Review by AlexandrZagorskiy — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (AlexandrZagorskiy) [reply README.md:59]: Ok, got it

**Pull request merged**